pkg/cedar/core/authority_to_operate.go

```diff
@@ -0,0 +1,86 @@
+package cedarcore
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/guregu/null/zero"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/models"
+
+	apiauthority "github.com/cmsgov/easi-app/pkg/cedar/core/gen/client/authority_to_operate"
+)
+
+// NOTE: This CEDAR endpoint in webMethods is called with a set of optional parameters (including a system ID) with the caveat that if
+//   a system ID is provided then all other provided parameters are ignored. This poses an interesting scenario for EASi b/c we will
+//   always be provide a system ID with our queries which effectively turns the system ID into a required parameter and nullifies/invalidates
+//   the other optional parameters. For this reason we are not going to include the optional parameters in the client methods for EASi.
+
+// GetAuthorityToOperate makes a GET call to the /authority_to_operate endpoint
+func (c *Client) GetAuthorityToOperate(ctx context.Context, cedarSystemID string) ([]*models.CedarAuthorityToOperate, error) {
+	if !c.cedarCoreEnabled(ctx) {
+		appcontext.ZLogger(ctx).Info("CEDAR Core is disabled")
+		return []*models.CedarAuthorityToOperate{}, nil
+	}
+
+	cedarSystem, err := c.GetSystem(ctx, cedarSystemID)
```

Comment:
There's an error in the system_summary.go code this calls, in `GetSystem()`; a bad if-condition throws the whole thing off. Give me ~30 minutes and I can push a quick fix to this branch.

Reply:
Fix pushed in https://github.com/CMSgov/easi-app/pull/1636/commits/92e11f4a6205cca6f7d228f0bf451d4bf9d92b74. Can someone besides me take a look at the changes I made there? @downeyn-cms @jstrang-oddball @ClayBenson94

Reply:
Fix LGTM - good catch Dylan.